.github/workflows/test.yml

```diff
@@ -0,0 +1,52 @@
+name: test
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: ${{ github.ref != 'refs/heads/main' }}
+
+jobs:
+  lint:
```

Comment:
nit: the order of the jobs is the order they are executed? license was executed before test in jenkins

Reply:
They run in parallel. To run sequentially each job will need a `needs` line with the previous job name